### ISSUE TITLE
question - census designated place population from get_estimates()

### ISSUE THREAD

**Issue by RadicalResearchLLC** [walkerke/tidycensus#594]
I am trying to pull census designated place population estimates using `get_estimates()` but I am only getting city/town level results.

I have tried geography = 'place' and 'county subdivision'.  

```
place_pop2 <- get_estimates(
  geography = "place",
  state = 'CA',
  vintage = 2023
)
```
This call works but it only returns 482 cities and towns in California.  A call to `tigris` and its `places()` function returns 1618 places.  CDPs with LSAD == 57 are excluded.
```
CA_places <- places(state = 'CA', cb = T, 
                    year = 2023)
```

Replacing geography = 'place' with 'county subdivision'  indicates that the geography is not available.  

Is this a census API issue?  I have not found any documentation that would suggest that CDP population estimates should not be returned.  My apologies if this doesn't follow full issue guidelines.

**Comment by walkerke** [comment]
Yeah, CDPs are not included in the Population Estimates - just incorporated places.
